src/lib/process/index.ts

```diff
@@ -55,9 +57,29 @@ export const createProcess = async (
   const polkadot: Polkadot.Polkadot = await createNodeApi(options)
   const expectedVersion: number = (await getVersion(polkadot, processId)) + 1
 
-  if (version !== expectedVersion) {
+  // TODO simplify
+  if (version < 1 || version > expectedVersion) {
```

Comment:
don't we want it to fail if the version is too low as well?

Reply:
we do `if (version < 1)` mean version smaller than 1 e.g. 0 -1 -2 -3 -4 -5 -5.... also if higher

Reply:
what if `version: 4` is on chain and we try to add a process with `version: 3`?

Reply:
I think is debatable as the error 'incrementing' and version < 1 kinda covers it. Will update

Reply:
it also has a unit test that covers it.
```
    it('fails to create for too low version', async () => {
      return assert.isRejected(
        createProcess(validProcessName, -1, validAllRestrictions, false, polkadotOptions),
        VersionError
      )
    })
    ```

Reply:
so not sure if I understand your statement...

Reply:
this is irrelevant now. Closing